### DISCUSSION TITLE
No Toolbar Icon

### DISCUSSION THREAD

**Discussion by MDJonas** [Cookie-AutoDelete/Cookie-AutoDelete#1626] [Q&A / Support]
I've just installed CAD on my Firefox browser:

- OS: aarch64 Mac OS (Please add OS version on paste)
- Browser Info: Firefox 126 (20240526221752)
- CookieAutoDelete Version: 3.8.2

There's no Icon on my Toolbar to open CAD....CAD is in my list of Extensions and is turned ON. What am I not doing since all my other Extensions are on the Toolbar?  Thank you for helping.

**Comment by chromes306** [comment]
I have the Same issue. 
Using  Mid 2010 Mac Pro running 10.13.6., Firefox 115.16.0ers, and there are no Menu icons for CAD, using CAD 3.8.2, and as you know, accessing the icon is how we configure CAD.  Please advise.

**Reply by MDJonas** [reply]
Thank you for the information....and for answering.

**Comment by kennethtran93** [comment]
In Firefox, is there an icon in the toolbar that looks like a puzzle icon?  That would be where any extensions that aren't pinned to the toolbar lives.  You may need to go into that puzzle icon to see the CAD icon, then click on the setting/gear icon next to the CAD extension and click 'Pin To Toolbar'

**Comment by chromes306** [comment]
That was the ticket.  Several others were hiding in that puzzle piece.
Thank you for the come back.
Michael L
********************************
<<<  Live S T R O N G  >>>
